src/app/pages/LiquidityMining/components/HistoryTable/TableBody/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { LiquidityPoolDictionary } from 'utils/dictionaries/liquidity-pool-dictionary';
+import { TableRow } from '../TableRow/index';
+
+export const TableBody: React.FC = () => {
+  const pools = LiquidityPoolDictionary.list();
+
+  return (
+    <tbody className="mt-5">
+      {pools.map((item, index) => (
+        <TableRow
+          key={`${item.supplyAssets[0].asset}/${item.supplyAssets[1].asset}/${index}`}
```

Comment:
If i understand correctly, you can shorten it by using item.poolAsset (it should be unique for each pool, at least for now)
```suggestion
          key={`${item.poolAsset}/${index}`}
```

Reply:
I wanted to be super sure it won't clash but it's true that it looks bit overengineered :D